stable/rabbitmq-ha/README.md

```diff
@@ -124,6 +124,7 @@ and their default values.
 | `extraVolumeMounts`                             | Extra volume mounts to mount to the statefulset                                                                                                                                                           | `[]`                                                     |
 | `rabbitmqEpmdPort`                             | EPMD port used for cross cluster replication                                                                                                                                                          | `4369`                                                     |
 | `rabbitmqErlangCookie`                         | Erlang cookie                                                                                                                                                                                         | _random 32 character long alphanumeric string_             |
+| `forceCookie`                                 | When this is true and rabbitmqErlangCookie is set, force on disk creation of the erlang cookie | `false`             |
```

Comment:
Remove `and rabbitmqErlangCookie is set,`

Reply:
done